lib/base/stream.cpp

```diff
@@ -83,7 +83,8 @@ void Stream::Close()
 
 	/* Force signals2 to remove the slots, see https://stackoverflow.com/questions/2049291/force-deletion-of-slot-in-boostsignals2
 	 * for details. */
-	OnDataAvailable.connect(std::bind(&StreamDummyCallback));
+	//OnDataAvailable.connect(std::bind(&StreamDummyCallback));
+	//TODO:: Replace SteamDummyCallback with something more purposeful
```

Comment:
Replace? I'd just

* [ ] wait until the Stream class disappears in master
* [ ] rebase

or

* [ ] not replace bind in this particular case